QKSMS/src/main/java/com/moez/QKSMS/ui/messagelist/MessageListFragment.java

```diff
@@ -474,7 +475,6 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 makeCall();
                 return true;
 
-            case R.id.menu_uw_convo_private_text:
```

Comment:
For sake of consistency, delete R.id.menu_uw_convo_private not R.id.menu_uw_convo_private_text. "_text" means that the option is appearing as text in the menu. Without the _text means that it is aimed to appear as an icon on the top. I believe that mark private is appearing twice because the screen can not take more icons so it pushes it to the menu. Could you somehow check if it is appearing on the top header or being pushed in the menu? Because we will have duplication issue for other options too if the screen size is small. Just for context, for each option we have placed it twice (one for icon and one for test in the menu) because we want to have those icons appear on the top for convenient marking but in case some user is novice, we have the option in the menu as text in case user don't remember which icon is for what. Android tries to puts option items with icon field set on the top header of the conversation. If it can't then it pushes it to the side menu as full text.

Sorry for long rambling here.